<?xml version="1.0"?>
<launch>
    <node pkg="tl_detector" type="tl_detector.py" name="tl_detector" output="screen" cwd="node">
        <param name="traffic_light_is_close" value="100.0" />        
        <param name="create_ground_truth" value="False" />        
        <param name="ground_truth_dir" value="data_gt" />        
        <param name="ground_truth_start_number" value="1" />        
        <param name="is_simulator" value="True" />        
        <param name="SVC_PATH" value="$(find tl_detector)/svc.p" />
        <param name="FCN_PATH" value="" />
    </node>        
</launch>
